mbtools/extract_html_content.py

```diff
@@ -14,7 +14,7 @@ def replace_content_tags(mbz_path, output_file_path,
         if isinstance(act, MoodleLesson) and "lesson" in filter \
                 or isinstance(act, MoodlePage) and "page" in filter:
             for html_elem in act.html_elements():
-                html_file = html_elem.replace_content_tag()
+                html_file = html_elem.replace_content_with_tag()
```

Comment:
This change seems extraneous / unnecessary for this PR now and can be backed out I think.